docs/src/main/sphinx/release/release-417.md

```diff
@@ -0,0 +1,40 @@
+# Release 417 (10 May 2023)
+
+## General
+
+* Improve performance of `UNION ALL` queries. ({issue}`17265`)
+* Fix over-reading data in the `TEXTFILE` decoder for structs that contain more
```

Comment:
This belongs in the connector that uses the TEXTFILE decoder. Also, it should be described in terms of user-visible behavior. Do users know what "TEXTFILE decoder" is?

Reply:
Probably Hive connector..